spec/features/associate_twitter_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe "Twitter association" do
+
+  context "can link to Twitter" do
+
+    before do
+      visit "/users/new"
+      fill_in "Email", with: "test@test.com"
+      fill_in "Password", with: "Password"
+      fill_in "Password confirmation", with: "Password"
+      click_button "Submit"
+    end
+
+    it "should be able to add a Twitter id" do
+      visit "/settings"
+
+      fill_in "Twitter ID", with: "1234"
+      click_button "Update"
+
+      expect(page).to have_content "Successfully updated settings"
+    end
+
+    it "should have the Twitter user id" do
+      #expect(page).to have_selector("input", exact: "1234")
```

Comment:
For this to work, you'll have to visit the settings page and fill in all the things, then click submit, like you did on lines 16-19 above.